generators/conduct/index.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const YeomanGenerator = require('yeoman-generator')
+
+class CommunityConductGenerator extends YeomanGenerator {
+  constructor (args, options) {
+    super(args, options)
+
+    this.option('email', {
+      defaults: this.options.email || this.user.git.email(),
+      desc: 'Contact email',
+      required: true,
+      type: String
+    })
+
+    this.option('generateInto', {
+      defaults: '',
+      desc: 'Destination directory',
+      required: false,
+      type: String
+    })
+  }
+
+  prompting () {
+    if (this.options.email) {
+      return Promise.resolve(true)
+    } else {
+      const prompts = [
+        {
+          name: 'email',
+          message: 'Contacts\'s e-mail',
+          when: !this.options.email,
+          default: this.user.git.email(),
+          store: true
+        }
+      ]
+      return this.prompt(prompts).then((answer) => {
+        this.options.email = answer.email
+      })
+    }
+  }
+
+  writing () {
+    this.fs.copyTpl(
+      this.templatePath('en/_CODE_OF_CONDUCT.md'),
+      this.destinationPath(this.options.generateInto, 'CODE_OF_CONDUCT.md'),
+      this.options
+    )
+  }
+};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1116)

Reply:
This statement is required in order to provide a contact email.